src/Wrapper.js

```diff
@@ -166,10 +172,14 @@ class Wrapper extends React.Component {
       walletWeb3,
       wrapper,
       appsLoading,
-      locator: { instanceId, params },
+      locator: { instanceId, params, dao },
+      isDaoDeprecated,
     } = this.props
+
     return (
-      <React.Fragment>
+      <ModalProvider>
+        <ModalView />
+        {isDaoDeprecated && <DeprecatedBanner dao={dao} />}
```

Comment:
Thoughts on moving both the modal and the deprecation banner up one level, to be in the [App](https://github.com/aragon/aragon/blob/master/src/App.js)? Right now the banner shows up during onboarding but it's not really obvious because it's here.

Reply:
I'll adhere to both your thoughts on this. @sohkai and @bpierre 👆